router/smoke/openshift/inventory-amq-interconnect-1-tls-auth.yml

```diff
@@ -13,8 +13,8 @@ all:
           router_config: /etc/qpid-dispatch/qdrouterd.conf
           implementation: dispatch
           service: qdrouterd
-          pem_file: client.crt
-          key_file: client-key.pem
+          pem_file: "client.crt"
```

Comment:
Next time I'm for using only one type of (quotation marks) > apostrophes in document/project.

Reply:
Done.